scripts/js/democracy/submitProposal.js

```diff
@@ -0,0 +1,22 @@
+const { ApiPromise, WsProvider, Keyring } = require("@polkadot/api")
+
+async function propose(proposalHash, deposit = 100_000_000_000_000) {
+    const provider = new WsProvider("ws://0.0.0.0:9944");
+    const api = await ApiPromise.create({ provider });
+    const keyring = new Keyring({ type: "sr25519" });
+    const sudo = keyring.addFromUri("//Alice");
+
+    await api.tx.democracy.propose(proposalHash, deposit)
```

Comment:
As of now, no. The process for taking a process from start to finish is fairly asynchronous and requires reacting to certain events emitted on chain. We could write a single process for this, but I think it will take more time than is worth it. Let me know if you still think its something worth pursuing.